internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -0,0 +1,163 @@
+package vm_console_proxy
+
+import (
+	"fmt"
+
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"kubevirt.io/ssp-operator/internal/common"
+	"kubevirt.io/ssp-operator/internal/operands"
+	vm_console_proxy_bundle "kubevirt.io/ssp-operator/internal/vm-console-proxy-bundle"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	operandName      = "vm-console-proxy"
+	operandComponent = "vm-console-proxy"
+)
+
+type vmConsoleProxy struct {
+	ServiceAccounts     []core.ServiceAccount
+	ClusterRoles        []rbac.ClusterRole
+	ClusterRoleBindings []rbac.ClusterRoleBinding
+	Services            []core.Service
+	Deployments         []apps.Deployment
+	ConfigMaps          []core.ConfigMap
```

Comment:
These fields are not used anywhere. They should be created and reconciled, instead of the hardcoded values in `resources.go`.

Reply:
And they don't need to be slices. For now, we only have one of each resource.

Reply:
Done, thanks! :)